### PR TITLE
Wk10 ecn20

### PR DESCRIPTION
**Pull request by emmanyhof**
Adding week 10 and 8 code/files (didn't realize I never pushed my week 8 stuff!)

### CODE REVIEW

**Comment by bbdaniels** [comment]
I don't see the README or any results for Week 8?

For Week 10, it's a good start, and you should finish the README with graphs and for Part 5.

I will mark both as incomplete for now.

File: Individual Assignments/Nyhof Emma/week-08/week-08-code.do
Comment (mz547) [inline Individual Assignments/Nyhof Emma/week-08/week-08-code.do:1]: Good work! I will suggest adding one biased model to see the variance in CIs and estimated coefficients.

File: Individual Assignments/Nyhof Emma/week-10/week-10-code.do [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+gen state = _n
+gen u_i = rnormal(1,3) // state effects
+gen unemp = rnormal(0.032,0.0085)
+bysort state: gen temprand_state = rnormal(0.5,0.1)
+expand int(runiform(10000,100000))
+gen u_ij = rnormal(1,2) // person effects
+bysort state: gen person = _n
+gen temprand_person = runiform()
+gen unemployed = runiform()<unemp // affects treatment but not outcome
+gen hoursworked = rnormal(25,5) if unemployed != 0 // confounder
```
Comment (mz547) [inline Individual Assignments/Nyhof Emma/week-10/week-10-code.do:18]: The confounder should both affect treatment and Y. Now the confounder 'unemployed' only affects Y.

File: Individual Assignments/Nyhof Emma/week-10/week-10-code.do [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
```
Comment (mz547) [inline Individual Assignments/Nyhof Emma/week-10/week-10-code.do:83]: Same issue as above. The confounder does not affect treatment.

File: Individual Assignments/Nyhof Emma/week-10/week-10-code.do [hunk 3]
```diff
@@ -0,0 +1,240 @@
...
+
+foreach x in `samples' {
+    append using `sims`x''
+}
+
+mean beta, over(N)
+
+
+
+* 5. DGP where random error terms are ONLY determined at the cluster level
```
Comment (mz547) [inline Individual Assignments/Nyhof Emma/week-10/week-10-code.do:221]: The codes in the do-file are incomplete.

File: Individual Assignments/Nyhof Emma/week-10/week-10-code.do [hunk 4]
```diff
@@ -0,0 +1,240 @@
...
+keep if hoursworked < 20 | unemployed == 1
+*egen temprand = rowmean(temprand_state temprand_person)
+gen treatment = temprand_person<0.5
+gen age = runiform(18,64) //
+
+gen wkly_inc = rnormal(800,250) ///
+			+ (100 * treatment) ///
+			+ (8 * hoursworked) /// 
+			+ (4.5 * age) ///
+			+ u_i + u_ij
```
Comment (mz547) [inline Individual Assignments/Nyhof Emma/week-10/week-10-code.do:178]: The random error should only occur at the cluster level.

**Review by mz547 — APPROVED** [review]
Good work! But there seem to be some issues in creating confounders and setting clustered random errors.

**Pull request merged**